content/post/release-of-1.4.0.md

```diff
@@ -0,0 +1,189 @@
+
+_Thanks to **[@lafriks](https://github.com/lafriks)**_
+
+## Responsive UI ([#2750](https://github.com/go-gitea/gitea/pull/2750))
+
+![Responsive UI screenshot](/demos/2750/1.png)
+
+![Responsive UI screenshot](/demos/2750/2.png)
+
+![Responsive UI screenshot](/demos/2750/3.png)
+
+Starting from version 1.4.0, Gitea's web interface is finally responsive; this means it can be properly used from your phone without having to pinch everywhere to see what is on the screen! There may be some places where the interface is still not completely responsive, in which case we ask you kindly to [file an issue](https://github.com/go-gitea/gitea/issues) if there isn't one already, posting screenshots of what does not work.
+
+_Thanks to **[@thehowl](https://github.com/thehowl)**_
+
+## Merge options in Pull Requests ([#3188](https://github.com/go-gitea/gitea/pull/3188))
+
+![Screenshot of merge options](/demos/3188/1.png)
+
+Hey, look! Someone submitted a pull request to your repo. Awesome. Except... it needs some work. And some more work. And some more. Just a tiny little bit... Done. Add also a few thousand merge-from-master commits along the way. And so you know that after the merge, your git history will never be the same again... that is, until today! Thanks to Squash and Merge, you can squash all the commits into a single one, and leave your git history stainless! We also added support for Rebase and Merge, similar to GitHub's merge options.
+
+_Thanks to **[@lafriks](https://github.com/lafriks)**_
+
+## Mention completion in issue editor ([#3136](https://github.com/go-gitea/gitea/pull/3136))
+
+![Mention completion demo](/demos/3136/1.gif)
+
+"Is this the right username?" is no more. With our shiny new mention completion, you can now start typing an username or a name of someone on your Gitea instance, and results will start popping up as you type.
+
+_Thanks to **[@harryxu](https://github.com/harryxu)**_
+
+## Progress bar for issues with checkboxes ([#3171](https://github.com/go-gitea/gitea/pull/3171))
+
+![Progress bar demo](/demos/3171/1.png)
+
+Did you know that Gitea has, like GitHub, support for checkboxes in markdown? If you write a bulletpoint in the following format: `- [ ]` you will get a nice HTML textbox. And does this have to do with checkboxes? Everything, of course! The issue list will now show the percentage of checked items in the top issue post. So you always know how far you've gotten on the roadmap.
+
+_Thanks to **[@modmew8](https://github.com/modmew8)**_
+
+## Long commits now have an expandable body ([#2980](https://github.com/go-gitea/gitea/pull/2980))
+
+![Commit body expansion demo](/demos/2980/1.gif)
+
+Making a long commit? Now you can view the entire contents of it without changing the page; you will only need to press on the ellipsis button, similar to what you can see on GitHub and GitLab.
```

Comment:
`Making a long commit` sounds like you put a lot of changes in it. Maybe `Did someone write an entire essay in the commit message?` 😄